README.md

```diff
@@ -1,27 +1,89 @@
 # @typescript/analyze-trace
-Tool for analyzing the output of `tsc --generateTrace` automatically, rather than following the steps [here](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).
+
+A tool for analyzing the output of `tsc --generateTrace` in a fast and digestible way (rather than more involved diagnostics [here](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).
 
 Note: The goal is to identify clear-cut hot-spots and provide enough context to extract a small repro.
 The repro can then be used as the basis of a bug report or a starting point for manual code inspection or profiling.
 
 ## Usage
 
-First, build your project with `tsc --generateTrace traceDir`.  This will create a new `traceDir` directory with paired trace and types files.
+The short version is to run these commands:
+
+```sh
+tsc -p tsconfig.json --generateTrace traceDir
+npm install --no-save @typescript/analyze-trace
+npx analyze-trace
+```
+
+Each of these commands do the following:
+
+1. First you will need to build your project with `--generateTrace` to a specific directory.
+   `tsc -p tsconfig.json --generateTrace traceDir` will create a new `traceDir` directory with paired trace and types files.
+   If your configuration file is not `tsconfig.json`, you will need to adjust the command.
+2. Installing `@typescript/analyze-trace` makes its various commands available in your project.
+3. Running `npx analyze-trace traceDir` outputs a sorted list of compilation hot-spots - places where TypeScript is taking a high amount of time.
 
-Next, install `@typescript/analyze-trace` to make its various commands available in your project.
+For best results, the analyzer should run on trace files that were generated in the same relative location.
```

Comment:
Maybe something like "If you run the analyzer from a location where the relative paths in the trace files resolve correctly, the analyzer will produce more helpful output."?